src/subclass/object.rs

```diff
@@ -700,6 +703,17 @@ mod test {
         assert_eq!(value.get::<String>(), Ok(Some("return value".to_string())));
     }
 
+    #[test]
+    fn test_callback_validity() {
+        let type_ = SimpleObject::get_type();
+        let obj = Object::new(type_, &[("name", &"old-name")]).expect("Object::new failed");
+
+        obj.connect_notify(Some("name"), |_, _| {
+            println!("hello!");
+        });
+        obj.notify("name");
```

Comment:
This one here is called as part of the `notify()` call, that's fine. Just check for it to actually happen.